tools/vsimporter/src/SBProject.cpp

```diff
@@ -163,6 +163,17 @@ bool SBProject::checkTargetCycle(const PBXTarget* target)
   }
 }
 
+bool SBProject::containsPackagebleProject()
+{
+  for (auto target : m_existingTargets) {
+    TargetProductType productType = target.second->getProductType();
+    if (productType == TargetStaticLib || productType == TargetBundle || target.first->getType() == "PBXAggregateTarget") {
```

Comment:
What do you mean? At import we can only tell what kind of xcode targets they are trying to import.

---
In reply to: [102370117](https://github.com/Microsoft/WinObjC/pull/2020#discussion_r102370117) [](ancestors = 102370117)